LLM Testing/Code-Testing/src/main/java/App.java

```diff
@@ -5,32 +5,33 @@
 import java.util.List;
 
 public class App {
-    public static void main(String[] args) throws IOException, GitAPIException {
+    public static void main(String[] args) throws IOException, GitAPIException, InterruptedException {
```

Comment:
Just wondering what the InterruptedException is for

Reply:
Forgot to remove it. Fixed.